wagtail/snippets/templates/wagtailsnippets/snippets/usage.html

```diff
@@ -1,47 +1,6 @@
-{% extends "wagtailadmin/base.html" %}
-{% load i18n %}
-{% block titletag %}{% blocktrans trimmed with title=object %}Usage of {{ title }}{% endblocktrans %}{% endblock %}
-{% block bodyclass %}model-{{ model_opts.model_name }}{% endblock %}
+{% extends "wagtailadmin/generic/index.html" %}
 
 {% block content %}
     {% include 'wagtailsnippets/snippets/headers/usage_header.html' %}
```

Comment:
This is so that the breadcrumbs can be shown.

Honestly I think we should revisit this, whether we want to keep the breadcrumbs for "internal views" (e.g. the usage and history views), or have the breadcrumbs stop at the edit view.

Currently, there's a disparity between pages and snippets: the breadcrumbs are shown on such views for snippets, but not for pages. When I implemented the new features for snippets, I added the header as it's necessary for the side panel toggles and thus the breadcrumbs came with it. I then took the initiative to also show the header on the internal views, and no one was against it, so now we have this.

However, the same hasn't been done for pages, so I think we should either remove the breadcrumbs or add the them for pages as well. It might not make sense for pages though, as the breadcrumbs are tied to the tree structure.

See also #9227.